ops/cloudbuild.cd.yaml

```diff
@@ -8,10 +8,17 @@ steps:
           --build-arg VITE_CLOUDINARY_API_KEY=$$VITE_CLOUDINARY_API_KEY \
           --build-arg VITE_CLOUDINARY_SECRET=$$VITE_CLOUDINARY_SECRET \
           --build-arg VITE_BIBLE_API_KEY=$$VITE_BIBLE_API_KEY \
+          --build-arg DO_OAUTH_TOKEN=$$DO_OAUTH_TOKEN \
```

Comment:
Looks like we do not use `DO_OAUTH_TOKEN` anymore. Can we remove it from the cloudbuild pipe and Dockerfile?